core/src/main/kotlin/com/malinskiy/marathon/report/allure/AllureReporter.kt

```diff
@@ -98,7 +98,7 @@ class AllureReporter(val configuration: Configuration, private val outputDirecto
                 mutableListOf(
                     ResultsUtils.createHostLabel().setValue(device.serialNumber),
                     ResultsUtils.createPackageLabel(test.pkg),
-                    ResultsUtils.createTestClassLabel(test.clazz),
+                    ResultsUtils.createTestClassLabel(suite),
```

Comment:
Can you please provide some docs on why test class should be specified using fqcn here and not just a simple class name?
Same for the full name above, would like to understand where is the source of truth for these

Reply:
@Litemn ping

Reply:
@Litemn another ping

Reply:
Here some example of test result generated by allure lib
[allure kotlin](https://github.com/allure-framework/allure-kotlin/blob/master/allure-kotlin-junit4/src/main/kotlin/io/qameta/allure/kotlin/junit4/AllureJunit4.kt#L139)
[allure java](https://github.com/allure-framework/allure-java/blob/b782b29f45e1720481f2c853eb9a03cd13bc11cc/allure-junit-platform/src/main/java/io/qameta/allure/junitplatform/AllureJunitPlatformUtils.java#L39)
<img width="608" alt="Screenshot 2022-04-22 at 15 04 33" src="https://user-images.githubusercontent.com/1678579/164711516-e9daa999-62f6-4fb3-a6fe-a3bb4759962c.png">
<img width="541" alt="example-junit-5-allure" src="https://user-images.githubusercontent.com/1678579/164711991-fa7c0ddf-4261-4054-9dad-cab5244237c1.png">

kotlin and java implemantation work same and use FQN for fullName and package for suite